src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/Atom10Constants.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace System.ServiceModel.Syndication
 {
-    public static class Atom10Constants
+    static class Atom10Constants
```

Comment:
I thought the standard was to be explicit in the accessibility, even when it's the default.